lib/diagnostic.rb

```diff
@@ -26,7 +26,16 @@
 # be writable.
 
 ##
-# your response here
+class Person
+  attr_reader :name :location
```

Comment:
`name` should be both readable and writable, so it should be declared with `attr_accessor`.  `location` should only be writable, so it should be declared with `attr_writable`.